src/providers/twitch/api/Helix.hpp

```diff
@@ -1120,6 +1166,13 @@ class Helix final : public IHelix
         ResultCallback<HelixChatters> successCallback,
         FailureCallback<HelixGetChattersError, QString> failureCallback);
 
+    // Get moderator list - This method is what actually runs the API request
+    // https://dev.twitch.tv/docs/api/reference#get-moderators
+    void fetchModerators(
+        QString broadcasterID, QString after,
```

Comment:
This function should have an `int first` parameter too like fetchChatters

The usages of this should probably just set this to 100